### PR TITLE
Vs/circles

### PR DESCRIPTION
**Pull request by VladimirSt13**
1) added decorative ellipses on the home page to:
- hero section
- customer section 
- junfolio section
2) minor fixes and refactor code

### CODE REVIEW

**Review by AnnaSidiakina — APPROVED** [review]

File: src/components/Home/HomeParts/HeroSection.jsx [hunk 1]
```diff
@@ -41,7 +43,7 @@ const textVariants = {
 
 const containerStyles = {
   position: 'relative',
-  overflow: 'hidden',
+  // overflow: 'hidden',
```
Comment (forze961) [inline src/components/Home/HomeParts/HeroSection.jsx:46]: Remove unused or add todo

File: src/components/Home/HomeParts/HeroSection.jsx [hunk 2]
```diff
@@ -93,12 +147,28 @@ const HeroSection=()=> {
           style={textContainerStyles}
         >
           <TextContainer>
-            <div>
+            <div style={{ position: 'relative' }}>
```
Comment (forze961) [inline src/components/Home/HomeParts/HeroSection.jsx:150]: If we using object styled components why we need to use Inline style?

File: src/components/Portfolio/Portfolio/Portfolio.jsx [hunk 3]
```diff
@@ -44,7 +44,7 @@ export const Portfolio = () => {
 
   return (
     <>
-      <Container>
+      {/* <Container>
```
Comment (forze961) [inline src/components/Portfolio/Portfolio/Portfolio.jsx:47]: We need a comment?

File: src/components/Home/HomeParts/JuniorsSection.jsx [hunk 4]
```diff
@@ -8,28 +8,224 @@ import {
...
+    if (!scope?.current) {
+      return;
+    }
+    const currentVariant =
+      direction === 'forward' ? variant.visible : variant.hidden;
+    fn(scope.current, currentVariant, variant.transition);
+  };
+
+  const handleAnimate = direction => {
+    handleAnimation(
```
Comment (forze961) [inline src/components/Home/HomeParts/JuniorsSection.jsx:159]: We can reuse this code. I see same code below.

**Review by forze961 — CHANGES_REQUESTED** [review]

File: src/components/Home/HomeParts/HeroSection.jsx (hunk 2 above)
Comment (VladimirSt13) [reply src/components/Home/HomeParts/HeroSection.jsx:150]: Added styled div

File: src/components/Portfolio/Portfolio/Portfolio.jsx (hunk 3 above)
Comment (VladimirSt13) [reply src/components/Portfolio/Portfolio/Portfolio.jsx:47]: deleted

File: src/components/Home/HomeParts/JuniorsSection.jsx (hunk 4 above)
Comment (VladimirSt13) [reply src/components/Home/HomeParts/JuniorsSection.jsx:159]: this code - is calling function with some parameters, blelow - calling the same function with other parameters. How to reuse this code?

Comment (forze961) [reply src/components/Home/HomeParts/JuniorsSection.jsx:159]: > this code - is calling function with some parameters, blelow - calling the same function with other parameters. How to reuse this code?

For example: Junior & Customer sections have very similar code.  Nice to have a a global section container where we can describe and then call  reusable functions or JSX. Then on index.js call <GlobalSection type=...>

File: src/components/Portfolio/Portfolio/Portfolio.jsx [hunk 5]
```diff
@@ -1,26 +1,26 @@
...
     setIsloading(true);
     try {
-      const { success = false, projects = [] } = await getData("portfolio");
+      const { success = false, projects = [] } = await getData('portfolio');
       if (!success) {
-        setErr("Something went wrong");
+        setErr('Something went wrong');
       }
       setProjectsList(projects);
     } catch (error) {
```
[the remark this answers is not in the document]
Comment (painlalala) [reply src/components/Portfolio/Portfolio/Portfolio.jsx:26]: ???

[the remark this answers is not in the document]
Comment (VladimirSt13) [reply src/components/Portfolio/Portfolio/Portfolio.jsx:26]: add alert message: Не вдалось отримати проекти

**Review by forze961 — APPROVED** [review]

**Pull request merged**